app/src/main/java/com/worldsnas/starplayer/model/MusicMapper.kt

```diff
@@ -0,0 +1,19 @@
+package com.worldsnas.starplayer.model
+
+import com.worldsnas.starplayer.api.MusicResponse
+import javax.inject.Inject
+
+class MusicMapper @Inject constructor() : Mapper<MusicResponse, MusicRepoModel> {
+    override fun mapToModel(type: MusicResponse): MusicRepoModel {
+        return MusicRepoModel(
```

Comment:
in faqat chetor provide mishe toye module?
va inke baraye online hast faqat!
mishe be interface function ezafe kard dige?